### PR TITLE
[openimageio] Update to version 2.2.7.0

### LINKED ISSUE #13621: [openimageio:x64-linux] build failure
**Host Environment**
- OS: Pop!_OS 20.04 LTS
- Compiler: gcc version 8.4.0 (Ubuntu 8.4.0-3ubuntu2) (Is this what goes here?)

**To Reproduce**
`./vcpkg install openimageio[core,ffmpeg,freetype,opencv,gif,openjpeg,webp,tools]`
This is the command I actually need to get working, the comment below this has all 128 variants of the command and if they worked or not. 

**Failure logs** 
-Cut and past the appropriate build messages from the console output.
-Please attach any additional failure logs mentioned in the console output.
[install-x64-linux-rel-out.log](https://github.com/microsoft/vcpkg/files/5250756/install-x64-linux-rel-out.log)
```
-- Building x64-linux-dbg
-- Building x64-linux-rel
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:141 (message):
    Command failed: /home/digitalkitsune/github/vcpkg/downloads/tools/cmake-3.17.2-linux/cmake-3.17.2-Linux-x86_64/bin/cmake --build . --config Release --target install -- -v
    Working Directory: /home/digitalkitsune/github/vcpkg/buildtrees/openimageio/x64-linux-rel
    See logs for more information:
      /home/digitalkitsune/github/vcpkg/buildtrees/openimageio/install-x64-linux-rel-out.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_build_cmake.cmake:91 (vcpkg_execute_build_process)
  scripts/cmake/vcpkg_install_cmake.cmake:24 (vcpkg_build_cmake)
  ports/openimageio/portfile.cmake:65 (vcpkg_install_cmake)
  scripts/ports.cmake:79 (include)


Error: Building package openimageio:x64-linux failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `./vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: openimageio:x64-linux
  Vcpkg version: 2020.06.15-unknownhash
```

**Additional context**
Add any other context about the problem here, such as what you have already tried to resolve the issue.

### LINKED ISSUE #13750: [openimageio] update to 2.2.6.1
Library name: openimageio

New version number: 2.2.6.1

Other information that may be useful (release notes, etc...)
[2.2.6.1](https://github.com/OpenImageIO/oiio/releases/tag/Release-2.2.6.1)
See the link for the long list of improvements that have been made in openimageio 2.2. 
I request that the openimageio port be updated to the current 2.2.6.1 release version.

Please let me know if there is something that I can do to help accelerate incorporation of this new version.

Thanks for your work on vcpkg--it does make my development work more productive!

### PR DESCRIPTION
**Pull request by brukted**
**Describe the pull request**

- What does your PR fix? Fixes #13750 Fixes #13621

- Which triplets are supported/not supported? All 
- Have you updated the CI baseline? No

- Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)? Yes

- This PR is based on #13808

### CODE REVIEW

File: ports/openimageio/CONTROL [hunk 1]
```diff
@@ -1,9 +1,9 @@
 Source: openimageio
-Version: 2.1.16.0
-Port-Version: 3
+Version: 2.2.7.0
+Port-Version: 4
```
Comment (Hoikas) [inline ports/openimageio/CONTROL:3]:
```suggestion
```

File: ports/openimageio/CONTROL [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 Homepage: https://github.com/OpenImageIO/oiio
 Description: A library for reading and writing images, and a bunch of related classes, utilities, and application
-Build-Depends: libjpeg-turbo, tiff, libpng, libheif, openexr, boost-thread, boost-smart-ptr, boost-foreach, boost-regex, boost-type-traits, boost-static-assert, boost-unordered, boost-config, boost-algorithm, boost-filesystem, boost-system, boost-thread, boost-asio, boost-random, robin-map, boost-stacktrace, fmt
+Build-Depends: libjpeg-turbo, tiff, libpng, libheif, openexr, boost-thread, boost-smart-ptr, boost-foreach, boost-regex, boost-type-traits, boost-static-assert, boost-unordered, boost-config, boost-algorithm, boost-filesystem, boost-system, boost-thread, boost-asio, boost-random, robin-map, boost-stacktrace, fmt, python2
```
Comment (Hoikas) [inline ports/openimageio/CONTROL:6]: Why do we now depend on python2? The feature `pybind11` implies that it builds the python bindings. Further, it seems inappropriate to build bindings for a sunsetted version of the python interpreter.

Comment (brukted) [reply ports/openimageio/CONTROL:5]: I have looked deeper and the linux build failures was caused because Its trying to build docs which requires python.

**Comment by LilyWangL** [comment]
@brukted Thanks for your PR. I test all features and `openimageio[pybind11]` build failed with the following error:
```
-- Found PythonInterp: F:/Lily/pr13966/vcpkg/downloads/tools/python/python-3.8.3-x86/python.exe (found version "3.8.3") 
CMake Error at F:/Lily/pr13966/vcpkg/installed/x86-windows/share/pybind11/FindPythonLibsNew.cmake:161 (message):
  Python libraries not found
Call Stack (most recent call first):
  F:/Lily/pr13966/vcpkg/scripts/buildsystems/vcpkg.cmake:496 (_find_package)
  F:/Lily/pr13966/vcpkg/installed/x86-windows/share/pybind11/pybind11Tools.cmake:16 (find_package)
  F:/Lily/pr13966/vcpkg/installed/x86-windows/share/pybind11/pybind11Config.cmake:100 (include)
  F:/Lily/pr13966/vcpkg/scripts/buildsystems/vcpkg.cmake:496 (_find_package)
  src/cmake/checked_find_package.cmake:81 (find_package)
  src/python/CMakeLists.txt:7 (checked_find_package)


-- Configuring incomplete, errors occurred!
```
And could you resolve conflicts?

**Comment by brukted** [comment]
@LilyWangL I will try to resolve the issue. It seems the problem is with pybind11/FindPythonLibsNew.cmake it self in fact there are a lot of issues about it on the  pybind11 repo.

**Comment by Hoikas** [comment]
While working on #13556, I found that the pybind11 python artifact specification is incorrect. They are referring to python 3.6 instead of python 3.8.

**Comment by brukted** [comment]
Waiting for #13556  to be merged.

**Comment by Hoikas** [comment]
@brukted That PR is being delayed for a more in depth review. It might be better to fix pybind11 here or in another PR. I don't mind resolving conflicts in my PR :smile:

**Comment by NancyLi1013** [comment]
@brukted 

Could you please resolve conflicts?

**Comment by Hoikas** [comment]
@brukted The pybind issue should have been fixed in #14510 :smile:

**Comment by NancyLi1013** [comment]
ping @brukted for response. 
Is work still being done for this PR?

**Comment by brukted** [comment]
@NancyLi1013 yes.

**Comment by NancyLi1013** [comment]
Why do you close this PR?

**Comment by brukted** [comment]
It is because I have to cover a whole semester in a month.

**Comment by NancyLi1013** [comment]
Thanks for your reply @brukted.

You can reopen this PR if you'd like to continue working on it.

**Pull request closed without merging**